src/rviz/default_plugin/path_display.cpp

```diff
@@ -499,11 +499,11 @@ void PathDisplay::processMessage( const nav_msgs::Path::ConstPtr& msg )
         const geometry_msgs::Point& pos = msg->poses[ i ].pose.position;
         Ogre::Vector3 xpos = transform * Ogre::Vector3( pos.x, pos.y, pos.z );
         axes_vect[i]->setPosition(xpos);
-        Ogre::Quaternion orientation(msg->poses[ i ].pose.orientation.w,
-                                     msg->poses[ i ].pose.orientation.x,
-                                     msg->poses[ i ].pose.orientation.y,
-                                     msg->poses[ i ].pose.orientation.z);
-        axes_vect[i]->setOrientation(orientation);
```

Comment:
Couldn't you simply use
`axes_vect[i]->setOrientation(transform * orientation);`
here?